react/README.md

```diff
@@ -0,0 +1,158 @@
+# Introduction
+
+This React sample demonstrates how to perform login actions that will initiate flow endpoint operations are used only to implement custom authentication UIs. 
+OIDC/OAuth 2 requests initiate the flow and redirect the browser to the custom authentication UI (which is configured in the application through the application’s `loginPageUrl` property.)
```

Comment:
Do people know what the custom authentication UI is at this point?

Reply:
Covered by [this comment](https://github.com/pingidentity/pingone-customers-sample-oidc/pull/5#discussion_r276996194)